src/app/tasks/create-task/create-task.component.ts

```diff
@@ -25,20 +30,35 @@ export class CreateTaskComponent {
     description: new FormControl(),
     type: new FormControl(null, Validators.required),
     createdOn: new FormControl(new Date().toDateString()),
-    status: new FormControl('In Progress', Validators.required)
+    status: new FormControl(null, Validators.required)
   });
 
-  constructor(private taskService: TaskService) {}
+  constructor(private taskService: TaskService) {
+    // Map the task type enum to array of keys and values
+    Object.keys(TaskType).forEach(taskTypeKey => {
+      this.taskTypes.push({
+        key : taskTypeKey, 
+        value : TaskType[taskTypeKey as keyof typeof TaskType]
+      });
+    });
+    // Map the task status enum to array of keys and values
+    Object.keys(TaskStatus).forEach(taskStatusKey => {
```

Comment:
Object.entries(TaskStatus).forEach(([key, value]) => {
      this.taskStatuses.push({
        key : key, 
        value : value
      });
    });